eslint.config.mjs

```diff
@@ -275,12 +273,38 @@ const config = createConfig([
     },
     settings: {
       tailwindcss: {
-        callees: ['twMerge', 'twClassName'],
+        callees: ['twMerge'],
         config: 'apps/storybook-react/tailwind.config.js',
         classRegex: ['^(class(Name)?|twClassName)$'],
       },
     },
   },
+  // Tailwind ESLint for React Native
```

Comment:
Adding the same rules for react native but taking in to account twrnc and the new tailwind config